R/functions_postestimate.R

```diff
@@ -0,0 +1,120 @@
+
+#' Extract model coefficients from estimate output
```

Comment:
Can we add tests for these methods?

Reply:
Done in [1c5ce54](https://github.com/snlab-ch/goldfish/pull/77/commits/1c5ce54d4414f4656aad101aa475fa9b7a2c80d0)